components/brave_shields/resources/panel/components/main-panel/style.ts

```diff
@@ -0,0 +1,134 @@
+import styled, { css } from 'styled-components'
+
+interface Props {
+  isOpen: boolean
+}
+
+export const Box = styled.div`
+  --pad-x: 22px;
+  background-color: ${(p) => p.theme.color.background02};
+  width: 100%;
+  height: 100%;
+
+  a {
+    font-family: Poppins;
+    color: ${(p) => p.theme.color.interactive05};
+    text-decoration: underline;
+  }
+`
+
+export const PanelHeader = styled.section`
+  width: 100%;
+  padding: 24px var(--pad-x) 24px var(--pad-x);
+  box-sizing: border-box;
+`
+
+export const ToggleBox = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-bottom: 15px;
+`
+
+export const PanelContent = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 0 25px 0;
+  position: relative;
+  z-index: 2;
+`
+
+export const SiteTitle = styled.h1`
+  color: ${(p) => p.theme.color.text01};
+  font-family: ${(p) => p.theme.fontFamily.heading};
+  font-size: 22px;
+  font-weight: 500;
+  letter-spacing: 0.02em;
+  margin: 0;
+  text-align: center;
+`
+
+export const AdvancedControlsButton = styled.button<Props>`
+  --border: 3px solid transparent;
+  --text-color: ${(p) => p.theme.color.text01};
+  --rotate: rotate(0deg);
+
+  background-color: ${(p) => p.theme.color.background03};
+  font-family: Poppins;
+  font-size: 14px;
+  color: var(--text-color);
+  width: 100%;
+  padding: 14px var(--pad-x);
+  border: var(--border);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  svg {
```

Comment:
This is assuming that the react components coming from the icon library are `svg`. Perhaps it would be more stable to use the common pattern of wrapping with an element to restrict the width / height and perform transform. Or if you really don't want multiple components then extending the style via `Styled(CaratStrongDownIcon)`, if that works.